rediscluster/client.py

```diff
@@ -1104,7 +1140,7 @@ def _random_good_hashslot_key(self, hashslot):
         Generate a good random key with a low probability of collision between any other key.
         """
         # TODO: Check if the key exists or not. continue to randomize until a empty key is found
-        random_id = "{{0}}{1}".format(hashslot, self._random_id())
+        random_id = "{{{0}}}{1}".format(hashslot, self._random_id())
```

Comment:
Also perhaps there's a conflation of hashslot and hashtag terminology here? Hashtags are the part within `{ }` according to the cluster spec. THis sin't relevant to the PR so I can just remove it.